Image Classifier Model/main/model.py

```diff
@@ -0,0 +1,62 @@
+from qwak.model.tools import run_local
```

Comment:
Sometimes deploying with run_local makes issues, just check that it works when deployed